public_desktop/desktop-app.js

```diff
@@ -89,6 +89,24 @@ function startDesktopApp() {
 			ev.preventDefault();
 		}
 	}
+	
+	var remote = require( 'electron' ).remote;
+	var buildEditorContextMenu = remote.require('electron-editor-context-menu' );
+	function contextMenu( ev ) {
+		if ( ! ev.target.closest( 'iframe body, textarea, input, [contenteditable="true"]' ) ) {
```

Comment:
Still in testing, will get removed. It doesn't work anyways - context menu doesn't show in iframes